### ISSUE TITLE
Wrong triggering rules selector-combinator-space-after in less

### ISSUE THREAD

**Issue by Tom910** [stylelint/stylelint#1681]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

It a bug, incorrectly triggered a rule in LESS

> Which rule, if any, is this issue related to?

selector-combinator-space-after

> What CSS is needed to reproduce this issue?

LESS
```less
.icon__border when (@size >= 60) and (@size < 102) {
    r: (@viewBox - @stroke2) / 2;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "selector-combinator-space-after": "always"
  }
}
```

> Which version of stylelint are you using?

6.8.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint test.less --syntax=less`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to LESS mixins.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
test.less
1:27  ✖  Expected single space after ">"   selector-combinator-space-after
```
